### ISSUE TITLE
Color Contrast Violation on Converge screen 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#292]
Describe the accessibility issue
Color contrast needs to be following WCAG guideline for contrast.  
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

To Reproduce
1. Open IL app
2. Go to Events
3. Click on " % match" link at the bottom of any event
4. "Converge" screen opens

Expected behavior
Colors on page should meet guideline ratio and do not. 
![Colour Contrast Analyser (CCA) 6_14_2021 4_07_07 PM](https://user-images.githubusercontent.com/82892873/121960230-6e22e300-cd2b-11eb-8fab-aa2fde123d2b.png)

Screenshots
![Colour Contrast Analyser (CCA) 6_14_2021 4_07_07 PM](https://user-images.githubusercontent.com/82892873/121960243-724f0080-cd2b-11eb-91f2-ab12812de58f.png)

Smartphone (please complete the following information):

![Your Phone 6_14_2021 4_05_36 PM](https://user-images.githubusercontent.com/82892873/121960353-9b6f9100-cd2b-11eb-8266-051077354818.png)
![Your Phone 6_14_2021 4_05_20 PM](https://user-images.githubusercontent.com/82892873/121960376-a0344500-cd2b-11eb-9cf0-54d9646742b7.png)

Samsung Galaxy S9 Android 10 
Additional context

**Comment by acovher2** [comment]
Hello 
It has been 2 months; are there any updates on this issue? 
Please provide any information you can., including whether this screen is even to be considered in scope with the app or not. 
Thank you.
